tracing/tests/event.rs

```diff
@@ -369,3 +369,38 @@ fn explicit_child_at_levels() {
 
     handle.assert_finished();
 }
+
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
+#[test]
+fn option_values() {
+    let (subscriber, handle) = subscriber::mock()
+        .event(
+            event::mock().with_fields(
+                field::mock("some_str")
+                    .with_value(&"yes")
+                    .and(field::mock("some_bool").with_value(&true))
+                    .and(field::mock("some_u64").with_value(&42_u64)),
```

Comment:
if we add a call to `.only()` here, it will additionally configure the test subscriber to assert that no other fields were recorded...which I think we probably want
```suggestion
                    .and(field::mock("some_u64").with_value(&42_u64))
                    .only(),
```